dags/vm_resource.py

```diff
@@ -158,3 +167,6 @@ class DockerImage(enum.Enum):
       "gcr.io/tpu-prod-env-multipod/maxtext_gpu_jax_nightly:"
       f"{datetime.datetime.today().strftime('%Y-%m-%d')}"
   )
+  MAXTEXT_ANISHA_TPU_JAX_STABLE = (
```

Comment:
We don't need this once your change is merged in MaxText right?

Reply:
Correct - removed - thanks!